### PR TITLE
5404 fix empty equipment category filter options

### LINKED ISSUE #5404: iSC/GAPS/Cold chain: Cold chain -> Equipment: Category filter field dropdown is not pre-populated
## What went wrong? 😲

Category filter field dropdown is not pre-populated (e.g. empty)

Screenshot 2024-11-14 at 4 37 41 PM

## Expected behaviour 🤔

Should be pre-populated for the users to select from, as you cannot type in the field

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Go to `Cold chain -> Equipment`
2. Click on the Equipment field dropdown
3. See empty output

## Your environment 🌱

- Open mSupply Version: `V2.4.0-RC1`
- Legacy mSupply Central Server Version: `V7.18.11`

- Platform: Google Chrome (browser on Mac) and Galaxy Tab A8 (Android)

- Database type: `SQLite3`

### PR DESCRIPTION
**Pull request by lache-melvin**
Fixes #5404

# 👩🏻‍💻 What does this PR do?

Category options now available:
Screenshot 2024-11-19 at 8 36 35 AM

Also fixes bug where if you add the `Type` filter, the options should be based on the selected category, but they weren't updating as you changed category.

## 💌 Any notes for the reviewer?

# 🧪 Testing

- [ ] Go to equipment list
- [ ] See category filter options - there should be 3!
- [ ] Select a category
- [ ] Add `Type` filter, and view the filter options
- [ ] Change the category filter
- [ ] Type filter options should update

# 📃 Documentation

- [ ] **Part of an epic**: documentation will be completed for the feature as a whole
- [x] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1.
  2.

### CODE REVIEW

**Comment by noel-yeldos** [comment]
Can confirm that the three category options are visible in the dropdown. Can also confirm that when type filter is added, the options available in the dropdown change depending on the category chosen.

**Comment by lache-melvin** [comment]
Thanks Noel! If your comment means you are happy for this code to be merged - can you approve the pull request? (The `review changes` button from the `files changed` tab) 🙏

**Comment by noel-yeldos** [comment]
Sorry, I've only reviewed the functionality! Someone else will need to review the code :)

**Review by roxy-dao — CHANGES_REQUESTED** [review]
LGTM~

Usually when we have a filter for something we display it in the columns as well... but we don't do that for the Category. Is that intentional...?

**Comment by lache-melvin** [comment]
I think just saving column space, probably discernible based on type - but you're right, have added 🙏

File: client/packages/coldchain/src/Equipment/ListView/ListView.tsx [hunk 1]
```diff
@@ -63,6 +63,14 @@ const AssetListComponent: FC = () => {
       width: 150,
       label: 'label.asset-number',
     },
+    {
+      key: 'categoryId',
```
Comment (roxy-dao) [inline client/packages/coldchain/src/Equipment/ListView/ListView.tsx:67]: categoryName?

**Review by roxy-dao — APPROVED** [review]
Awesome! Thanks for the changes (:

**Pull request merged**